### PR TITLE
A simple Profiler class.

### PR DESCRIPTION
**Pull request by PiezPiedPy**
A simple profiler and some helper functions placed in Util.cs to support the profiler, they are also handy for use in the rest of the code base.

The profiler is enabled/disabled with the compiler directive TRAJECTORIES_PROFILER.

Use the Start and Stop methods to wrap the code you want to measure.

Start KSP, load a game and press Ctrl+p to open the profiler window.

### CODE REVIEW

**Comment by fat-lobyte** [comment]
Looks good! Please rebase, resolve conflicts and we're good to go.

**Comment by PiezPiedPy** [comment]
ready for merge ;)

File: Plugin/Trajectories.csproj [hunk 1]
```diff
@@ -1,124 +1,125 @@
...
-if $(ConfigurationName)==Release (call "$(ProjectDir)..\misc\VisualStudio\buildscripts\ZipPack\build-package.bat" $(TargetName) -v@(VersionNumber))</PostBuildEvent>
-  </PropertyGroup>
-  <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
-       Other similar extension points exist, see Microsoft.Common.targets.
-  <Target Name="BeforeBuild">
-  </Target>
-  <Target Name="AfterBuild">
-  </Target>
-  -->
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (fat-lobyte) [inline Plugin/Trajectories.csproj]: What's up with the whitespace changes? Diff looks like you rewrote the whole file ;)

Comment (PiezPiedPy) [inline Plugin/Trajectories.csproj]: GitHub must of added them, I only changed one line lol :/

**Comment by PiezPiedPy** [comment]
Restarting this PR

**Pull request closed without merging**